feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -59,4 +51,35 @@ internal class HomeViewModel @Inject constructor(
             }
         }
     }
+
+    private fun initialize() {
+        updateState {
+            copy(
+                mapBoxPublicKey = buildConfigProvider.mapBoxPublicKey,
+                mapBoxStyleUri = buildConfigProvider.mapBoxStyleURI,
+                areFeedingPointsLoading = false
```

Comment:
I don't see usages for `areFeedingPointsLoading` variable

Reply:
It was in case some progress bar was needed. I'll remove it.